RELEASE_NOTES.md

```diff
@@ -0,0 +1,45 @@
+# cddl-gen v. 0.3.0
+
+## Improvements:
+
+### CDDL language
+ * Add more support for hexadecimal/octal/binary numbers in CDDL. E.g. in size ranges or value limits
+ * Add support for ranges with three dots in CDDL files
+ * Change CDDL quantifier syntax from x**y to x*y
```

Comment:
Since this is markdown file anyway, I'd suggest using the inline "\`" marker for these lines, since markdown will take asterisks as either bold or italics, like it did here in the render between with "x**y to x*y" instead of `x**y` to `x*y`.